### DISCUSSION TITLE
Stacking layout column width

### DISCUSSION THREAD

**Discussion by lgestc** [leftwm/leftwm#736] [Q&A] [answered]
Hey, it seems that stacking layout columns are split evenly. Is it possible to have it setup with different proportions, eg. 3:2?

**Comment by VuiMuich** [comment, marked as answer]
We have the `IncreaseMainWidth`/`DecreaseMainWidth` commands to adjust the main window.
And we plan to have a more sophisticted layout system in the future.

**Reply by lgestc** [reply]
Thank you ;)
